ec2utils/ec2uploadimg/lib/ec2utils/upload_VERSION

```diff
@@ -1 +1 @@
-5.0.0
+5.0.1
```

Comment:
Consider having this be a minor bump, as you're adding a new feature (specifying VPC subnet in config files).

Reply:
The major bump is from inserting the billing_codes argument into the ctor of the imaplementation which makes the API incompatible